src/components/screens/Authenticator/modules/CodeRequest/CodeRequest.interactor.vue

```diff
@@ -71,7 +71,13 @@ export default {
 
         this.$emit('complete');
       } catch (err) {
-        this.error = this.$i18n.t('components.code.authFailed');
+        switch (err.code) {
+          case 409:
+            this.error = this.$i18n.t('components.code.userAlreadyExists');
+            break;
+          default:
+            this.error = this.$i18n.t('components.code.authFailed');
```

Comment:
`break` required, if this is not prettier

Reply:
Why break in `default` statement?

Reply:
For consistency only, I think this is good habit 🙂 because although here there isn't code after switch (which would be executed if it were here), it may be added in future, or you may miss `break` after default out of habit somewhere. So technically, this is not required, but I personally put `break` always (only `return` or `throw` are exceptions). Just imo.

Reply:
don't need in default